tasks/Debian.yml

```diff
@@ -1,15 +1,26 @@
----
-- name: Add specific key
-  apt_key: id={{docker_repo_key}} keyserver=keyserver.ubuntu.com state=present
 
-- name: Install https apt transport package
-  apt: pkg=apt-transport-https state=present update_cache=yes
+- name: Add specific key
+  apt_key: keyserver={{docker_repo_key_server}} id={{docker_repo_key}}
 
 - name: Add docker repo
-  apt_repository: repo='deb {{ docker_repo }} docker main' state=present
+  apt_repository: repo='deb {{docker_repo}} ubuntu-{{ansible_distribution_release}} main' state=present
```

Comment:
just for the previous comment,  `{{ ansible_distribution | lower }}-{{ ansible_distribution_release }} ` is OK instead , isnt it? FYI my patchset is ceaaf073810af67f9efbc3745e34ee00b580e7e1